centreon/tests/e2e/features/Resources-status/01-listing/index.ts

```diff
@@ -185,6 +185,91 @@ Then(
   }
 );
 
+Given('a saved critical service filter', () => {
+  cy.fixture('resources/criticalServicesFilter.json').then((filters) =>
+    setUserFilter(filters)
+  );
+
+  cy.visit('centreon/monitoring/resources').wait([
+    '@getFilters',
+    '@monitoringEndpoint'
+  ]);
+
+  cy.contains('Unhandled alerts').should('be.visible');
+
+  cy.get(`div[data-testid="selectedFilter"]`).click();
+
+  cy.contains('Critical_Services');
+});
+
+When('I select the critical service filter', () => {
+  cy.contains('Critical_Services').click();
+});
+
+Then(
+  'only the critical services are displayed in the result',
+  () => {
+    cy.contains(serviceInDtName).should('be.visible');
+    cy.contains(secondServiceInDtName).should('be.visible');
+  }
+);
+
+Given('a saved pending host filter', () => {
+  cy.fixture('resources/pendingHosts.json').then((filters) =>
+    setUserFilter(filters)
+  );
+
+  cy.visit('centreon/monitoring/resources').wait([
+    '@getFilters',
+    '@monitoringEndpoint'
+  ]);
+
+  cy.contains('Unhandled alerts').should('be.visible');
+
+  cy.get(`div[data-testid="selectedFilter"]`).click();
+
+  cy.contains('Pending_Hosts');
+});
+
+When('I select the pending host filter', () => {
+  cy.contains('Pending_Hosts').click();
+});
+
+Then(
+  'only the pending hosts are displayed in the result',
+  () => {
+    cy.contains('host1').should('be.visible');
```

Comment:
this does not assert all hosts are pending
you should find all the elements in the list and check if everything is pending